add.go

```diff
@@ -517,7 +517,7 @@ func (b *Builder) Add(destination string, extract bool, options AddAndCopyOption
 			if sourceIsGit(src) {
 				go func() {
 					var cloneDir, subdir string
-					cloneDir, subdir, getErr = define.TempDirForURL(tmpdir.GetTempDir(), "", src)
+					cloneDir, subdir, dirErr = define.TempDirForURL(tmpdir.GetTempDir(), "", src)
```

Comment:
This should just `return` if `getErr` is not `nil`, as there's no point in continuing, and `cloneDir` and `subdir` are not expected to be useful values.